layers/+tools/transmission/README.org

```diff
@@ -63,52 +64,68 @@ Add =(transmission :variables transmission-auto-refresh-all t)= to
+| ~SPC m s p~ | Set bandwidth priority of torrent(s) at point or in region.       |
+| ~SPC m s d~ | Set global download speed limit in kB/s.                          |
+| ~SPC m s l~ | Set label(s) for marked torrent(s) or torrent at point.           |
+| ~SPC m s r~ | Set global seed ratio limit.                                      |
+| ~SPC m s u~ | Set global upload speed limit in kB/s.                            |
+| ~SPC m t m~ | Toggle mark on torrent(s) at point or in region.                  |
+| ~SPC m t s~ | Toggle torrent between started and stopped.                       |
+| ~SPC m t t~ | Toggle =transmission-turtle-mode=.                                |
 
 ** =transmission-info-mode=
 
 | Key binding | Description                                                   |
 |-------------+---------------------------------------------------------------|
 | ~SPC m c~   | Copy magnet link of current torrent.                          |
-| ~SPC m s d~ | Set download limit of torrent(s) at point in kB/s.            |
-| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point. |
-| ~SPC m s l~ | Set seed ratio limit of torrent(s) at point.                  |
 | ~SPC m m~   | Move torrent at point or in region to a new location.         |
-| ~SPC m a a~ | Add announce URLs to torrent or torrents.                     |
 | ~SPC m T~   | Remove trackers from torrent at point by ID or announce URL.  |
-| ~SPC m s u~ | Set upload limit of torrent(s) at point in kB/s.              |
+| ~SPC m a t~ | Add announce URLs to torrent or torrents.                     |
+| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point. |
+| ~SPC m s d~ | Set download limit of torrent(s) at point in kB/s.            |
+| ~SPC m s l~ | Set seed ratio limit of torrent(s) at point.                  |
 | ~SPC m s p~ | Set bandwidth priority of torrent(s) at point or in region.   |
+| ~SPC m s u~ | Set upload limit of torrent(s) at point in kB/s.              |
 
 ** =transmission-files-mode=
 
 | Key binding | Description                                                   |
 |-------------+---------------------------------------------------------------|
-| ~SPC m g f~ | Visit the file at point with =find-file-read-only=.           |
 | ~SPC m r~   | Run a command on the file at point.                           |
-| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point. |
+| ~SPC m g f~ | Visit the file at point with =find-file-read-only=.           |
 | ~SPC m g i~ | Open a =transmission-info-mode= buffer for torrent at point.  |
-| ~SPC m m~   | Move torrent at point or in region to a new location.         |
+| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point. |
+| ~SPC m m m~ | Move torrent at point or in region to a new location.         |
 | ~SPC m m u~ | Mark file(s) at point or in region as unwanted.               |
```

Comment:
for wanted and unwanted, we can use `SPC m t w/u` instead.
this frees up `SPC m m` for `transmission-toggle-mark`.

Reply:
That's a good idea! This is an easy change to make.

Reply:
Changed the prefixes for these 2 bindings and `SPC m m` is reserved for `transmission-toggle-mark` to maintain consistency with the other modes.